.pre-commit-config.yaml

```diff
@@ -0,0 +1,16 @@
+repos:
```

Comment:
I'd rather not have this pre-commit file. We have `poe lint` and I'd like that to be the primary checker. We enforce it on PR, otherwise devs can run it when they want.

You can add `blacken-docs` to that, but you might not want to because I don't expect once we have real Python docs that we will have any hand-written code blocks in any documentation inside this repository.